cli/azd/test/recording/sanitize.go

```diff
@@ -56,8 +56,8 @@ func sanitizeContainerAppListSecrets(i *cassette.Interaction) error {
 					// Redis requirepass. Sanitize the password, remove other config.
 					if strings.Contains(val, "requirepass ") {
 						body.Value[i].Value = to.Ptr("requirepass SANITIZED")
+						continue
```

Comment:
@weikanglim - I think this was subtly wrong before and the `continue` to was preventing us from sanitizing things unrelated to the `requirepass` entry. I think the code now matches the intention of the comment but I wanted to bring this to your attention.

Reply:
I had to make this change because the end to end tests ends up fetching the container secrets as we try to do our PATCH style update of the container apps image and the list of secrets included the registry password.

Reply:
This is right; there was a change in a previous PR for this fix that was reverted. Just for reference: https://github.com/Azure/azure-dev/commit/e4a70d0e5141de8c2823ae88bc053a311ef3d7c1#diff-4e16e271f78e59f4ae2cebd907f7c53aeb186b6a2f3a0adf8fec9de1664cd62a